compiler/damlc/daml-doc/src/DA/Daml/Doc/HaddockParse.hs

```diff
@@ -368,16 +374,20 @@ getTemplateDocs DocCtx{..} typeMap = map mkTemplateDoc $ Set.toList dc_templates
 -- | Extracts all names of Template instances defined in a module and a map of
 -- template to set of its choices (instances of Choice with a particular
 -- template).
-getTemplateData :: ParsedModule -> (Set.Set Typename, MS.Map Typename (Set.Set Typename))
+getTemplateData :: ParsedModule ->
```

Comment:
`<nit-pick>` haddock description is not correct any more

Reply:
that's a good point :-)

Reply:
Fixed!